### DISCUSSION TITLE
Is Kubernetes ingress controller - v3.0.0 docker image FIPS compliant?

### DISCUSSION THREAD

**Discussion by rishabhjain62** [Kong/kubernetes-ingress-controller#5259] [Q&A] [answered]
Is Kubernetes ingress controller - v3.0.0 docker image FIPS compliant? If not how can I build to make it FIPS compliant.

**Comment by programmer04** [comment, marked as answer]
The Docker image available in the registry is not FIPS compliant. For FIPS compliant images please [contact sales](https://konghq.com/contact-sales).

> _If the above answers your question please click `Mark as answer` button to increase visibility for others._
